### PR TITLE
Added standard_exponential to Numpy Frontend

### LINKED ISSUE #21021: standard_exponential

### PR DESCRIPTION
**Pull request by Preeti-ing**
close #21021

### CODE REVIEW

**Comment by a0m0rajab** [comment]
Thanks for the contribution, can you add the `close #issue_number` tag in the PR body? this will help us to auto close the issue and connect them together. 

Feel free to read this [GitHub docs ](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)in case if you need help.

**Comment by a0m0rajab** [comment]
Another thing, you have not written the test for your function, can you look at that too? you can find the related docs in [this docs](https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html)

**Comment by Preeti-ing** [comment]
> Another thing, you have not written the test for your function, can you look at that too? you can find the related docs in [this docs](https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html)

Hey! I have done both things that you asked. Please review. Also I am not able to find the re-review option here.

**Review by a0m0rajab — COMMENTED** [review]
Thanks for doing that, there I just wanted to clarify that I am not a team member in ivy though. 
For the review: 

- [ ] you need to add the close tag in the PR body (mentioned before)
- [ ] you need to have the functions in the correct file, you can rely on this docs: [where to place a frontend function](https://unify.ai/docs/ivy/overview/contributing/open_tasks.html#where-to-place-a-frontend-function)

**Comment by Preeti-ing** [comment]
Can anyone please review here? @a0m0rajab  @RickSanchezStoic

**Comment by RickSanchezStoic** [comment]
Hi @Preeti-ing seems like you undid your changes, can you confirm once by checking the diff? Thanks!

**Comment by Preeti-ing** [comment]
> Hi @Preeti-ing seems like you undid your changes, can you confirm once by checking the diff? Thanks!

The branch had some conflicts. I have made the changes, you can find them now.

File: ivy/functional/frontends/numpy/random/functions.py [hunk 1]
```diff
@@ -119,8 +119,7 @@ def multinomial(n, pvals, size=None):
 
 @to_ivy_arrays_and_back
 @from_zero_dim_arrays_to_scalar
-def negative_binomial(n, p, size=None):
-    if p <= 0 or p >= 1:
```
Comment (a0m0rajab) [inline ivy/functional/frontends/numpy/random/functions.py:123]: Questions, why did we remove this function?

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py [hunk 2]
```diff
@@ -507,6 +507,7 @@ def test_numpy_negative_binomial(
     fn_tree,
     backend_fw,
     on_device,
+
```
Comment (a0m0rajab) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py:510]: we can remove this line.

Comment (Preeti-ing) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py]: But I am working on standard_exponential not on negative_binomial

Comment (a0m0rajab) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py]: Nit: Yes, but some unnecessary changes have been made here. It's okay it's just nit.

Comment (Preeti-ing) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py]: What should I do now?

**Comment by Ishticode** [comment]
The issue is marked as complete and there is no activity on this PR for over 6 weeks.

**Pull request closed without merging**